README.md

```diff
@@ -90,7 +90,7 @@ After creating your application on the [Yoti Dashboard](https://www.yoti.com/das
 
 The variables required for the SDK to work are found in the tabs on your Yoti application's settings page ([Yoti Dashboard](https://www.yoti.com/dashboard/)). These are:
 
-* **`YOTI_APPLICATION_ID`** - This is used to configure the [Yoti Login Button](https://www.yoti.com/developers/documentation/#2-front-end-integration).
+* **`YOTI_SCENARIO_ID`** - This is used to configure the [Yoti Login Button](https://www.yoti.com/developers/documentation/#2-front-end-integration).
```

Comment:
Although it takes you to the right page through a redirect, the section is slightly different now that we've switched over the documentation: 
```suggestion
* **`YOTI_SCENARIO_ID`** - This is used to configure the Yoti Login Button (see [Front End Integration](https://developers.yoti.com/yoti-app-integration/yoti-app-integration#step-3-front-end-integration)).
```